packages/atomic/cypress/integration/segmented-facet-scrollable.cypress.ts

```diff
@@ -0,0 +1,117 @@
+      .init();
+  }
+  describe('verify rendering', () => {
+    before(setupScrollable);
+    ScrollableAssertions.assertDisplayScrollable(true);
+  });
+  describe('with overflowing segmented facets', () => {
+    function setupWithOverflowFacets() {
+      new TestFixture()
+        .with(addScrollable({field: 'author', 'number-of-values': 50}))
+        .init();
+    }
+
+    describe('verify rendering', () => {
+      before(setupWithOverflowFacets);
+      ScrollableAssertions.assertDisplayScrollable(true);
+      ScrollableAssertions.assertDisplayArrows(false, true);
+    });
+
+    describe('with right click', () => {
+      function setupClickArrowScrollable() {
+        setupWithOverflowFacets();
+        clickArrow('right');
+      }
+      before(setupClickArrowScrollable);
+      ScrollableAssertions.assertDisplayArrows(true, true);
+    });
+
+    describe.skip('with right scroll', () => {
+      function setupScrollScrollable() {
+        setupWithOverflowFacets();
+        scroll('right', 2000);
+      }
+      before(setupScrollScrollable);
+      ScrollableAssertions.assertDisplayArrows(true, true);
+    });
+
+    describe.skip('with right scroll using keyboard', () => {
+      function setupKeyboardScrollable() {
+        setupWithOverflowFacets();
+        cy.get(scrollableComponent).click('center');
+        cy.get(segmentedFacetComponent)
+          .shadow()
+          .find('[part="value-box"][aria-pressed="true"]')
+          .type('{rightarrow}');
+      }
+      before(setupKeyboardScrollable);
+      ScrollableAssertions.assertDisplayArrows(true, true);
+    });
+
+    describe('with screen size increase', () => {
+      before(() => {
+        setupWithOverflowFacets();
+        cy.viewport(7680, 4320);
+      });
+      ScrollableAssertions.assertDisplayScrollable(true);
+      ScrollableAssertions.assertDisplayArrows(false, false);
+    });
+  });
+  describe('without overflowing segmented facets', () => {
+    function setupWithoutOverflowFacets() {
+      new TestFixture()
+        .with(addScrollable({field: 'author', 'number-of-values': 4}))
+        .init();
+    }
+
+    describe('verify rendering', () => {
+      before(setupWithoutOverflowFacets);
+      ScrollableAssertions.assertDisplayScrollable(true);
+      ScrollableAssertions.assertDisplayArrows(false, false);
+    });
+
+    describe('with screen size decrease', () => {
+      before(() => {
+        setupWithoutOverflowFacets();
+        cy.viewport(320, 480);
+      });
+
+      ScrollableAssertions.assertDisplayScrollable(true);
+      ScrollableAssertions.assertDisplayArrows(false, true);
+    });
+  });
+
+  describe('with invalid segmented facets', () => {
+    function setupWithInvalidFacets() {
+      new TestFixture()
+        .with(addScrollable({field: 'invalidd', 'number-of-values': 4}))
+        .init();
+    }
+    before(setupWithInvalidFacets);
+    ScrollableAssertions.assertDisplayScrollable(false);
```

Comment:
are there any console errors in this case? if so, it might be nice to check that they are being logged. it looks like console functions are stubbed under `TestFixture.consoleAliases.error/log/warn`:

```js
    cy.window().then((win) => {
      cy.stub(win.console, 'error').as(
        TestFixture.consoleAliases.error.substring(1)
      );
      cy.stub(win.console, 'warn').as(
        TestFixture.consoleAliases.warn.substring(1)
      );
      cy.stub(win.console, 'log').as(
        TestFixture.consoleAliases.log.substring(1)
      );
    });
```

Reply:
There aren't any console errors.